sentences/fr/_common.yaml

```diff
@@ -308,9 +308,11 @@ expansion_rules:
   dans: "(dans|du|de)"
   de: "(du|de)"
   yatil: "(y a-t-il|il y a|y a t il|y a t'il|y a-t'il)"
+  sontils: "(a t|est|sont)[-][ ][(il[s]|elle[s])]"
```

Comment:
`a-t'il` is different than `est-t'il`. Why do we have the same expansion rule for that ?
Also, can't we use `estil` to have singular form. It's weird to name the expansion rules with plural name as it supports singular and plural.

Reply:
I had in mind "auxiliaires" verbs handling. That's maybe a lazy path, your remark makes sense.
I can split in:
<estil>: "(est|sont)[-][ ][(il[s]|elle[s])]"
and
<atil>: "(a([-| )t]|ont)[-][ ]((il[s]|elle[s])]"

According to Larousse, we should always write "a-t-il" and "est-il" with hyphens but I don't know how speech-to-text solutions deal with hyphens in French. 

That's also why I created <yatil> moreover, we can have "Y a-t-il une fenêtre ouverte ?" or "il y a une fenêtre ouverte ?"  in a more familiar way. 

What do you think?